packages/atomic/src/locales.json

```
@@ -5573,5 +5573,25 @@
     "zh": "弹出式菜单以获取更多选项卡",
     "zh-cn": "用于更多选项卡的弹出式菜单",
     "zh-tw": "用於更多標籤的快顯式選單"
+  },
+  "generated-answer-title": {
+    "en": "Generated answer for you",
+    "fr": "Réponse générée pour vous"
+  },
+  "this-answer-was-helpful": {
+    "en": "This answer was helpful",
+    "fr": "Cette réponse m'a aidé"
```

```suggestion
    "fr": "Cette réponse m'a été utile"
```
Juste pour éviter les controverses avec le participe passé...

---

Pas de probleme mais je suis curieux de quelles controverses tu parles 😅